router/dataplane.go

```diff
@@ -3301,10 +3215,8 @@ func (p *scionPacketProcessor) egressInterface( /*@ ghost oldPkt io.IO_pkt2 @*/
 // @ requires  acc(p.scionLayer.Mem(ub), R2)
 // @ requires  p.scionLayer.ValidPathMetaData(ub)
 // @ requires  p.path == p.scionLayer.GetPath(ub)
-// @ requires  acc(&p.buffer, R50) && p.buffer.Mem()
-// @ requires  acc(&p.buffWithFullPerm)
+// @ requires  acc(&p.buffer, R50) && p.buffer != nil && p.buffer.Mem()
```

Comment:
same as before. I guess this holds for every method that had `p.buffWithFullPerm` as a precondition before, and which does not have `Bytes` as a precondition